### ISSUE TITLE
Option to wrap preformatted text blocks

### ISSUE THREAD

**Issue by nervuri** [skyjake/lagrange#635]
Take this page as an example:

gemini://gemi.dev/gemini-mailing-list/messages/000190.gmi

The long lines don't fit on my screen and Lagrange doesn't seem to have a way to scroll to the right. Reading the full message is impossible, unless I right-click > "View as Plain Text" (with the "wrap long plain text lines" preference enabled).  Even with horizontal scrolling, reading long lines would be annoying.

Please add a an option to wrap preformatted text blocks (maybe a per-block option would make the most sense).

As for horizontal scrolling, I see there are already two open issues requesting this.

**Comment by skyjake** [comment]
I agree it would be useful to have an option to force-wrap a chosen preformatted block (i.e., overriding the preformatting), particularly on mobile.
